app/models/project.rb

```diff
@@ -6,7 +6,7 @@ class Project < ActiveRecord::Base
 
   has_many :first_choice_application_drafts,  class_name: 'ApplicationDraft', foreign_key: 'project1_id'
   has_many :second_choice_application_drafts, class_name: 'ApplicationDraft', foreign_key: 'project2_id'
-  has_many :teams # working on the project; after application procedure
+  has_many :accepted_teams, class_name: 'Team' # working on the project; after application procedure
 
```

Comment:
The good news: with the name `accepted_teams`, the comment is much more clear. But I feel it still needs the comment. For me, 'accepted_teams' is about the application procedure, less about 'working on the project'. 
I considered `:performing_teams`. ?